borgmatic/commands/borgmatic.py

```diff
@@ -616,10 +617,24 @@ def collect_configuration_run_summary_logs(configs, arguments):
         if 'extract' in arguments or 'mount' in arguments:
             validate.guard_single_repository_selected(repository, configs)
 
-        validate.guard_configuration_contains_repository(repository, configs)
+        if 'config' not in arguments:
+            validate.guard_configuration_contains_repository(repository, configs)
     except ValueError as error:
         yield from log_error_records(str(error))
         return
+    
+    if 'bootstrap' in arguments:
+        # no configuration file is needed for bootstrap
+        local_borg_version = borg_version.local_borg_version({}, 'borg')
+        borgmatic.actions.bootstrap.run_bootstrap(arguments['bootstrap'], arguments['global'], local_borg_version)
+        yield logging.makeLogRecord(
+            dict(
+                levelno=logging.INFO,
+                levelname='INFO',
+                msg='Bootstrap successful',
+            )
+        )
+        return
```

Comment:
Given that `bootstrap` can only really be used by itself, I wonder if it's worth erroring if there are other actions on the command-line. (Probably that wouldn't go here, but in `parse_arguments()` or similar.)

Reply:
Should that check be added after we decide what other sub commands should be added to the `borgmatic config` action? `parse_arguments()` seems to check for existence of other keywords, but we don't have any other than bootstrap.

Reply:
Ah okay gotcha (I didn't even know you could use multiple actions at once). So there should probably be a list of actions somewhere so I can do something like:

```python
    for action in action_names:
        if 'bootstrap' in arguments and action in arguments['bootstrap']:
            raise ValueError(
                'The bootstrap action cannot be used together with other actions.'
            )
```
?

Reply:
I think it may be even easier than that. How about:

```python
if 'bootstrap' in arguments.keys() and len(arguments.keys()) > 1:
    raise ValueError(...)
```

This would be before the `'global'` key gets set into `arguments` though.

> I didn't even know you could use multiple actions at once

It's handy for use cases like overriding the default borgmatic actions but just omitting one of them.

Reply:
Thanks, this makes sense!